src/SDKs/SqlManagement/Sql.Tests/AzureRmSqlServerActiveDirectoryAdministratorTest.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Resources.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Sql.Tests
+{
+    public class AzureRmSqlServerActiveDirectoryAdministratorTest
+    {
+        [Fact]
```

Comment:
@jaczhan please record test using latest nugets (e.g. 1.5.0 version of Resource manager) Currently you are using 1.1.0-preview version of Resource Manager.
All new tests should be using latest version of Resource Manager.